### ISSUE TITLE
Poor Performance with bigbird-roberta-base

### ISSUE THREAD

**Issue by awasthiabhijeet** [OhadRubin/SmBop#35]
Hi @OhadRubin ,

Replacing Grappa based encoder with bigbird-roberta-base, yields validation accuracy of just 40.8%.
Is this expected because best_validation_spider is not pre-trained using the Grappa-based objectives?

Thanks,

**Comment by OhadRubin** [comment]
Hey,
Yes, this is expected. SmBoP was hyperparameters were tuned w.r.t GraPPa, if you change a major part of the model like the encoder, you need to perform HP tuning again.
